gpu-aware-scheduling/cmd/gas-scheduler-extender/main.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"flag"
+
+	"github.com/intel/telemetry-aware-scheduling/extender"
+	"github.com/intel/telemetry-aware-scheduling/gpu-aware-scheduling/pkg/gpuscheduler"
+	"k8s.io/klog/v2"
+)
+
+func main() {
+	var (
+		kubeConfig, port, certFile, keyFile, caFile string
+		unsafe                                      bool
```

Comment:
Interesting - I don't think I've seen this style of var declaration done inside a function scope much